mm-bot/.env.example

```diff
@@ -11,3 +11,4 @@ SN_PRIVATE_KEY=
 HERODOTUS_API_KEY=
 LOGGING_LEVEL=
 LOGGING_DIR=
+ETHERSCAN_API_KEY=
```

Comment:
Why is this in the env for the MM bot. The etherscan api is only used for the deployment of ethereum contracts

Reply:
I added it there and forgot to move it to the .env file in the solidity folder. It's fixed now!